### PR TITLE
PR for code review of the project private_event

### PR DESCRIPTION
**Pull request by uchennaanya**
# What we did:
- Create custom user login
- Create custom controller actions
- Create forms
- Create user session
- Create advanced association and relationship between models and associations

### CODE REVIEW

**Review by rindrajosia — CHANGES_REQUESTED** [review]
**Changes Required** 

Hi there!!
Great job so far
But before moving on there are some changes that we need to work on.
REQUIRED:
- [x] Even if I am logged in, your APP asks me to `sign in ` when I click on `back` many times:

As you can see here, I am already `logged in` as `rindrajosia`.

- [x] `UsersController/show` action: **Lists events created by user**, i.e. @created_events = current_user.events ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/current_user_show_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

- [x] Add a `menu` in your app that helps `the user navigate`:

Here, for example, you can add a link that redirects to the page where the user can create an event

- [ ]You should add an index for `user_id` column (`events table`)

- [x] Remove inline comments

- [x] Add `Test unit` for your project.

1. models have unit tests for validations and associations (Rspec) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/unit_tests_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49723079/))
2. authentication have has integration tests (Rspec + Capybara) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/integration_tests_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49723079/))
3. event managment features have integration tests (Rspec + Capybara) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/integration_tests_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49723079/))

Guys work on the requested changes above and request another review.
Good luck.
Rindra Josia

File: README.md [hunk 1]
```diff
@@ -1,24 +1,62 @@
...
-* Database creation
+To get a local copy of the repository please run the following commands on your terminal:
 
-* Database initialization
+$ cd  folder-name
 
-* How to run the test suite
+~~~bash
+git clone https://github.com/kunmi02/private_event.git
+$ cd members_only
```
Comment (enaburekhan) [inline README.md]: This should be cd private_event

**Review by enaburekhan — CHANGES_REQUESTED** [review]
# status: Request changes:full_moon:
**Hi team!
Great job on implementing the changes requested by the previous Tse :clap::clap:. However, we just need to make a few changes to get this project approved:muscle:.**
**Mandatory requirements:**

- [ ] Navigation links are present at the top of the page ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/navigation_links_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

- [ ] the events table has column `creator_id` of type `integer`

- [ ] an index for `creator_id` column is added
- [ ] correct member_only to private_event in the Getting started in the readme.

- [ Optional] 
Make the UI to look more user friendly by adding some stylings.

Make the above required changes and when you are done, kindly request for another review.

You can reach me on slack @Eric Enaburekhan if you have any questions
Happy Coding:blush:
Eric Enaburekhan

**Review by NtwaliHeritier — APPROVED** [review]
### STATUS: APPROVED✔
Hello team, 
great work on this project.

As a tip, next time use more detailed commit messages.

**Pull request merged**